src/main.rs

```diff
@@ -18,11 +18,11 @@ async fn main() -> ExitCode {
 
 /// Simple entrypoint without any bells or whistles
 #[allow(clippy::print_stderr)]
-async fn run_sync() -> anyhow::Result<()> {
+async fn run_sync() -> Result<()> {
+	// Load CONFIG
 	let config = {
 		let config_path = std::env::var("FAMEDLY_LDAP_SYNC_CONFIG").unwrap_or("config.yaml".into());
```

Comment:
While we're doing breaking changes to the configuration, we might wanna take the opportunity to change the env var namespace from `FAMEDLY_LDAP_SYNC` to just `FAMEDLY_SYNC` or something

Reply:
Good point.

What about changing the package / repo name? At what point will we update `ldap-sync` to `famedly-sync`?

Reply:
So I am reverting this based on the Matrix discussion then.

Reply:
Here is a new issue for this topic: https://github.com/famedly/ldap-sync/issues/58